hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -205,6 +206,7 @@ public CacheConfig(Configuration conf, ColumnFamilyDescriptor family, BlockCache
     this.blockCache = blockCache;
     this.byteBuffAllocator = byteBuffAllocator;
     LOG.info("Created cacheConfig: " + this + (family == null ? "" : " for family " + family) +
```

Comment:
IMO better we should move this LOG from here to HStore. 
protected void createCacheConf(final ColumnFamilyDescriptor family) {
    this.cacheConf = new CacheConfig(conf, family, region.getBlockCache(),
        region.getRegionServicesForStores().getByteBuffAllocator());
  }
After the cacheConf is created.  CacheConfig is having toString() with all details.  This way we can avoid passing the RegionInfo/regionName unnecessarily to CacheConfig just for log.
In Master branch, we are not seeing the CacheConfig log with every compaction because of a different constructor being used.

Reply:
Yes, you are right, thank you for your suggestions